src/main/java/ch/njol/skript/util/ColorRGB.java

```diff
@@ -74,6 +74,11 @@ public static ColorRGB fromString(String string) {
 		);
 	}
 
+	public static Color fromBukkitOrRgbColor(org.bukkit.Color color) {
```

Comment:
This method is confusingly named. It only accepts bukkit colors but the name implies it accepts some non-bukkit rgb value too.
It should just be `fromBukkitColor()` like SkriptColor